test/parallel/test-runner-run.mjs

```diff
@@ -4,6 +4,9 @@ import { join } from 'node:path';
 import { describe, it, run } from 'node:test';
 import { dot, spec, tap } from 'node:test/reporters';
 import assert from 'node:assert';
+import { EventEmitter } from 'events';
+
+EventEmitter.setMaxListeners(35);
```

Comment:
This is a code smell, it should not be needed. Why did you add it?

Reply:
I added it during my local tests and forgot to remove it.
I've seen that this test prints a warning related to the maximum number of listeners.
This happens even without this change.

I'll remove it with the next commit

Reply:
removed